### PR TITLE
Correct some Esperanto translations

### PR DESCRIPTION
**Pull request by quarthex**

### CODE REVIEW

**Comment by Wuzzy2** [comment]
@quarthex: Rebase needed (and probably an update, too). The next release is planned for July 31, 2022.

**Comment by quarthex** [comment]
I finished the rebase.

**Comment by appgurueu** [comment]
There are still conflicts and the diff has gotten massive (many unrelated changes, rebasing went wrong). Please address this.

File: mods/default/locale/default.eo.tr [hunk 1]
```diff
@@ -167,14 +167,14 @@ Key=Ŝlosilo
...
 100% (output full)=100% (eligo estas plena)
 @1%=@1%
-Not cookable=Ne povas kuiri
+Not cookable=Nekuirebla
 Empty=Malplena
 Furnace active=Forno laboras
 Furnace inactive=Forno ne laboras
 (Item: @1; Fuel: @2)=(Objekto: @1; Brulaĵo: @2)
 Furnace=Forno
 Title:=Titolo
```
Comment (appgurueu) [inline mods/default/locale/default.eo.tr:176]: What about the colons here? Are colons used differently in Esperanto?

**Review by appgurueu — APPROVED** [review]

**Comment by appgurueu** [comment]
Please resolve the merge conflicts.

File: mods/default/locale/default.eo.tr (hunk 1 above)
Comment (quarthex) [reply mods/default/locale/default.eo.tr:176]: Are you talking about the missing one on the last line?
Because it is effectively a miss to be corrected.

**Comment by quarthex** [comment]
I am currently trying to merge it.
Let me some times.

File: mods/default/locale/default.eo.tr (hunk 1 above)
Comment (appgurueu) [reply mods/default/locale/default.eo.tr:176]: Shouldn't "Title:" be translated as "Titolo:" is what I'm trying to say (same for the line below that)

**Review by sfan5 — APPROVED** [review]
Can't check the contents ofc but looks okay.

**Pull request merged**